config/initializers/passwordless.rb

```diff
@@ -0,0 +1,4 @@
+Passwordless.default_from_address = "CHANGE_ME@example.com"
```

Comment:
We should probably ask the customer for an address to use here.

Reply:
Captured in trello task